appveyor.yml

```diff
@@ -11,6 +11,7 @@ environment:
   matrix:
     - language: cpp
       BUILD_DLL: ON
+      UNICODE: ON
```

Comment:
One config takes ~7 minutes. It seems we can afford more configurations. Travis-CI takes much longer (>30 minutes) anyway.
Is Unicode much more common than Ansi mode? If not, we might want to keep coverage of Ansi mode as well.
One data point: Chromium uses Unicode mode.